target/linux/pistachio/modules.mk

```diff
@@ -175,3 +175,49 @@ endef
 
 $(eval $(call KernelPackage,nhc_udp))
 
+define KernelPackage/uccp420wlan
+  SUBMENU:=$(WIRELESS_MENU)
+  TITLE:=Imagination wireless driver
+  KCONFIG:=CONFIG_UCCP420WLAN
+  KCONFIG:=\
+    CONFIG_UCCP420WLAN=y \
+    CONFIG_CFG80211=y \
```

Comment:
If this is set what happens to the kmod-cfg80211 module? Seems like we will have cfg80211 built in to the kernel plus the compat-wireless cfg80211 module.